controllers/userStatus.js

```diff
@@ -41,8 +41,8 @@ const getUserStatus = async (req, res) => {
     if (userId) {
       const userData = await userStatusModel.getUserStatus(userId);
       const { userStatusExists, id, data } = userData;
-      let responseObject = { id, userId, data: null };
-      if (data) responseObject = { ...responseObject, data };
+      const responseObject = { id, userId, data: null, message: null };
```

Comment:
Nitpick. 
Can the message be an empty string than null?

Example: message: ''

Reply:
yeah, empty string will be better. will update. Thanks.